index.js

```diff
@@ -1,88 +1,120 @@
 // ***** Question 1 *****
+
+let counter = 1
+counter = 2
+
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
+console.log("%cQuestion 1", "color: red") 
 
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+console.log(counter) 
+// => 2
+console.log("%c----------", "color: red") 
 
 
 // ***** Question 2 *****
 // *** Uncomment the lines below to test
 // *** after testing, comment the line causing the error back in 
 // *** otherwise, the error will stop the rest of your code from running
-// console.log("%cQuestion 2", "color: red") 
+const name = "Alex"
 
-// console.log(name)
+console.log("%cQuestion 2", "color: red") 
+
+console.log(name)
 // => "Raffy"
 // name = "Not Raffy" 
 // => TypeError
-// console.log("%c----------", "color: red") 
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 3 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
+function drinkWater(currentThirstLevel) {
+  console.log("Man I sure am thirsty")
+  currentThirstLevel -=1
+  console.log("Ahh that hits the spot")
+
+  return currentThirstLevel
+}
+
+console.log("%cQuestion 3", "color: red") 
```

Comment:
Great! You can also condense this in the following way:

`function drinkWater(thirstLevel) {
  console.log("Man I sure am thirsty");
  console.log("Ahh that hits the spot");
  return thirstLevel -= 1;
}`

Reply:
that makes sense